packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -1,7 +1,7 @@
 function setSelinux {
     source (loop0)/etc/os-release
     set baseSelinuxCmd=""
-    if regexp "rockylinux|fedora|almalinux|redhat|tumbleweed" $KAIROS_ARTIFACT; then
+    if test $KAIROS_FAMILY == "rhel" -o test $ID == "opensuse-tumbleweed"; then
```

Comment:
this one?

> this comment will be confusing after we merge this PR. Anyone viewing this file won't understand why we explain the non-usage of regexp.

Reply:
ah no, this one I guess:

> We can stick to KAIROS_ prefixed variables, which we fully control, if you use $KAIROS_FLAVOR_RELEASE == "tumbleweed" instead of ID.

Reply:
sorry, yes. These are both comments I made :D .